### ISSUE TITLE
Unable to add Amazon Music on the UK

### ISSUE THREAD

**Issue by callahad** [janbar/noson-app#48]
I'm in the UK. Amazon Music works great in the Sonos app, but I'd like to add it to Noson.

1. Services -> Add -> Amazon Music
2. See message "This will require to authenticate against the music service again, as credentials cannot be retrieved from Sonos device."
3. Click "Start service registration"
4. Click the auth link, which redirects to `https://na.account.amazon.com/ap/oa?clientContext=...`
5. Click "Allow", Amazon reports success
6. Noson reverts to the message from step 2.

While doing this, I see the following in the console:

```
(SONOS)ParseMessage: parse xml failed
qml: Retry request auth ...
qml: Retry request auth ...
qml: Retry request auth ...
(SONOS)ParseMessage: parse xml failed
```

**Comment by callahad** [comment]
No, I still get stuck in a loop where I authorize in the browser, then Noson asks me to go back and authorize again, over and over.

Using Noson 3.7.2.

*Should* this work right now, or are services still a bit rough around the edges? (I haven't looked at the code at all, yet.)

**Comment by janbar** [comment]
Some services using authentication "device link" won't work: google....
The app has been tested successfully with Spotify , Deezer. Also all services using authentication by login/password should work.

**Comment by janbar** [comment]
If the amazon registration is free, I will try it.

**Comment by janbar** [comment]
The user-agent had to be updated.
Fix by commit 641d6d4c4ab9cf61f6a7003430f6bf32c1a48375
